backend/FwLite/FwLiteWeb/appsettings.sdk.json

```
@@ -6,7 +6,8 @@
     "OpenBrowser": false
   },
   "FwDataBridge": {
-    "ProjectsFolder": "./fw-projects"
+    "ProjectsFolder": "./fw-projects",
+    "TemplatesFolder": "."
```

This would have been my gut choice.
```suggestion
    "TemplatesFolder": "./Templates"
```

---

I thought of that, but it's always just going to be empty, so there's not much point